src/fixtures/review.ts

```diff
@@ -20,6 +20,10 @@ import {
   MOCK_GITHUB_COMMIT_AUTHOR_TWO,
   MOCK_GITHUB_DATE_ONE,
   MOCK_COMMIT_FILEPATH_PLACEHOLDER,
+  MOCK_COMMIT_FILENAME_ONE,
+  MOCK_COMMIT_FILEPATH_ONE,
+  MOCK_COMMIT_FILENAME_TWO,
+  MOCK_GITHUB_DATE_TWO,
 } from "./identity"
```

Comment:
The tests in your description should allow the on-call to test step by step before deployment. Sample here https://github.com/isomerpages/isomercms-backend/pull/1173